client/payment-details/payment-method/afterpay-clearpay/index.tsx

```diff
@@ -4,21 +4,23 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
+import React from 'react';
 
 /**
  * Internal dependencies.
  */
 import Detail from '../detail';
+import { Charge } from 'wcpay/types/charges';
+import { PaymentMethodDetails } from 'wcpay/payment-details/types';
 
 /**
  * Extracts and formats payment method details from a charge.
  *
- * @param {Object} charge The charge object.
- * @return {Object}       A flat hash of all necessary values.
+ * @param {Charge} charge The charge object.
+ * @return {PaymentMethodDetails} A flat hash of all necessary values.
  */
-const formatPaymentMethodDetails = ( charge ) => {
+const formatPaymentMethodDetails = ( charge: Charge ): PaymentMethodDetails => {
```

Comment:
This function is also defined in `client/payment-details/payment-method/affirm/index.tsx` and appears exactly the same AFAICT. Could we maybe move this to a utility function somewhere else, to avoid duplicating code?